bindings/py/cpp_src/bindings/algorithms/py_TemporalMemory.cpp

```diff
@@ -180,9 +179,9 @@ using namespace nupic::algorithms::connections;
 
         py_HTM.def_property_readonly("extra", [](const HTM_t &self) { return self.extra; } );
 
-	py_HTM.def_property_readonly("anomaly", [](const HTM_t &self) { return self.anomaly; }, 
-	  "Anomaly score updated with each TM::compute() call. "	
-	);
+        py_HTM.def_property_readonly("anomaly", [](const HTM_t &self) { return self.anomaly; },
+          "Anomaly score updated with each TM::compute() call. "
```

Comment:
off-topic for this PR: we should add the doc that calling activateCells() instead of compute() will break your anomaly score. Or is the "updated with each compute()" clean enough?